benefits/core/templates/core/includes/media-list.html

```diff
@@ -6,11 +6,14 @@
                 <h3 class="media-body--heading">{{ item.heading }}</h3>
                 <div class="media-body--details">
                     <p>{{ item.details }}</p>
-                    <div class="media-body--items">
-                        <ul>
-                            {% for l in item.bullets %}<li>{{ l }}</li>{% endfor %}
-                        </ul>
-                    </div>
+                    {% if item.bullets|length %}
+                        <div class="media-body--items {{ item.bullets }}">
```

Comment:
What is this extra class using `{{ item.bullets }}`?

Reply:
That was left behind from some debugging. Just removed it